ietf/doc/mails.py

```diff
@@ -103,6 +103,36 @@ def email_stream_changed(request, doc, old_stream, new_stream, text=""):
               dict(text=text,
                    url=settings.IDTRACKER_BASE_URL + doc.get_absolute_url()),
               cc=cc)
+    
+def email_stream_state_changed_cfa(request, doc, cfa_duration=2):
+    (to, cc)= gather_address_lists('doc_stream_state_edited', doc=doc)
```

Comment:
This doesn't reach the recipients requested in the #9142 - it likely needs its own mailtrigger (see https://datatracker.ietf.org/mailtrigger/name/doc_stream_state_edited/ and https://datatracker.ietf.org/mailtrigger/name/).
Adding those will require a migration - let us know if you're already familiar with django migrations - if not, we can add them.